### ISSUE TITLE
Configure custom rulesets in .editorconfig

### ISSUE THREAD

**Issue by paulsamuels** [pinterest/ktlint#2939]
## Expected Behavior
It would be useful if you could specify the path/s to custom rulesets in the .editorconfig. Currently to get my project to use custom rulesets I've had to make a custom wrapper that the team uses to invoke `ktlint`. Although this works it's not as integrated and it's easy for someone to invoke `ktlint` directly without loading the custom rulesets, often resulting in a failure being raised at some point later in the CI pipeline.

## Additional information
* Current version of ktlint: 1.5.0

**Comment by paul-dingemans** [comment]
> Currently to get my project to use custom rulesets I've had to make a custom wrapper that the team uses to invoke ktlint.

What do you mean with this? How do you invoke ktlint? What build system are you using?

**Comment by paulsamuels** [comment]
Ideally I'd just invoke `ktlint` on the command line. What I was meaning above was I currently have to write a wrapper like

`ktlintw`
```bash
#!/bin/bash

ktlint -R ktlint-custom-rules/ktlint-custom-rules.jar "$@"
```

This is all just from the terminal. We do also have a gradle task set up, but the start up time of gradle tasks is long enough to make it not ideal to use.

**Comment by paul-dingemans** [comment]
Ok clear. Adding a configuration for this to the `.editorconfig` could be a solution for your specific usage with ktlint CLI. But it would be unclear for users that use a different integration with the KtlintRuleEngine. Tools like Detekt provide a very specific set of rules to Ktlint, and do not use the concept of rulesets as far as I know. Having an `.editorconfig` property to set custom ktlint rulesets in the `.editorconfig` would than communicate an functionality that will not work in their use case.

Do you use different custom rulesets per project? If not, it might be simpler to provide your ktlint wrapper at a shared location.

Also, you might want to consider to use the ktlint-intellij-plugin (requires usage of an JetBrains IDEA, or Android Studion), as it provides instant formatting of your rules. It allows you to configure custom rulesets per project. But it has the same downside that you have to configure that per project.
